src/aleph_client/asynchronous.py

```diff
@@ -584,6 +586,109 @@ async def create_program(
     )
 
 
+async def create_instance(
+        account: Account,
+        rootfs: str,
+        rootfs_size: int,
+        rootfs_name: str,
+        environment_variables: Optional[Mapping[str, str]] = None,
+        storage_engine: StorageEnum = StorageEnum.storage,
+        channel: Optional[str] = None,
+        address: Optional[str] = None,
+        session: Optional[ClientSession] = None,
+        api_server: Optional[str] = None,
+        sync: bool = False,
+        memory: Optional[int] = None,
+        vcpus: Optional[int] = None,
+        timeout_seconds: Optional[float] = None,
+        persistent: bool = False,
+        volumes: Optional[List[Mapping]] = None,
+        subscriptions: Optional[List[Mapping]] = None,
+) -> Tuple[InstanceMessage, MessageStatus]:
+    """
+    Post a (create) INSTANCE message.
+
+    :param account: Account to use to sign the message
+    :param rootfs: Root filesystem to use
+    :param rootfs_size: Size of root filesystem
+    :param rootfs_name: Name of root filesystem
+    :param environment_variables: Environment variables to pass to the program
+    :param storage_engine: Storage engine to use (Default: "storage")
+    :param channel: Channel to use (Default: "TEST")
```

Comment:
Yes, I agree with that, we can use None by default, but anyway I don't understand exactly what's the use case to use channels.

Reply:
I find it kind of weird TBH, if we can leave Channel `None`. When having `None` as a valid input, I would expect a drastically different behavior on these messages than on others. Meaning that:
1) They would never be filtered, if a filter would be applied
2) They could never be filtered, as otherwise you'd have to pass in a `List`, mixed with `str` and `NoneType` into the node software
3) Could only appear if no channel filter is applied at all

When thinking about scaling, this makes me uncomfortable. What would be the primary method of sharding messages then? Sender addresses?